iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -19,6 +19,7 @@
 #include "iceoryx_utils/cxx/optional.hpp"
 #include "iceoryx_utils/error_handling/error_handling.hpp"
 #include "iceoryx_utils/internal/units/duration.hpp"
+#include "iceoryx_utils/posix_wrapper/timer.hpp"
```

Comment:
Why is this line added?

Reply:
posix::Timer::now() is used in this file. It just happened that the timer.hpp was included by a different include, which is now not the case anymore.